language-support/ts/daml-types/index.ts

```diff
@@ -31,6 +31,10 @@ export interface Serializable<T> {
  */
 export interface Template<T extends object, K = unknown, I extends string = string> extends Serializable<T> {
   templateId: I;
+  /**
+   * @internal
+   */
+  sdkVersion: '0.0.0-SDKVERSION';
```

Comment:
I’m confused, this is hardcoded not the actual SDK version, how does this work?

Reply:
when we build the package with our bazel rules, all strings '0.0.0-SDKVERSION' in the source are replaced by the actual sdk version.

Reply:
Have you checked this? My understanding was that we only replace things in `package.json` not the source files. Looking at the Bazel rules seems to confirm that.

Reply:
ah nvm, we replace in all files.

Reply:
this should also resolve the above issue in the test, right?

Reply:
Not quite sure, we do this in `pkg_npm` not during the build. I think the tests might still be affected. Easy enough to test with `DAML_SDK_RELEASE_VERSION`.